frontend/src/views/category/Category.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <v-container fluid>
+    <content-card v-if="category()" toolbar back>
+      <template #title>
+        <v-toolbar-title class="font-weight-bold">
+          <CategoryChip :category="category()" dense large />
+          {{ category().name }}
+        </v-toolbar-title>
+      </template>
+
+      <template #title-actions>
+        <v-menu v-if="isManager" offset-y>
+          <template #activator="{ on, attrs }">
+            <v-btn icon v-bind="attrs" v-on="on">
+              <v-icon>mdi-dots-vertical</v-icon>
+            </v-btn>
+          </template>
+          <v-list>
+            <DialogEntityDelete
+              :entity="category()"
+              :warning-text-entity="category().name"
+              :dialog-title="$tc('views.category.category.deleteCategory')"
+              @submit="goToActivityAdmin"
```

Comment:
This also goes to activityAdmin if the delete Request fails.
It would be good if the user would be informed of the error
(but maybe this is a problem of DialogEntityDelete)

Reply:
I've added the redirect to a new success handler (additional functionality), because otherwise it throws errors, because the category doesn't exist anymore.

Reply:
It isn't quite there, i will create a new issue

[Screencast from 02.01.2024 10:27:18.webm](https://github.com/ecamp/ecamp3/assets/1506818/6b8b3393-7bd7-468a-8dfe-5b7f087b0aa2)

Reply:
Maybe this needs to be solved in the router